backend/Recodo.BLL/Services/UserService.cs

```diff
@@ -113,11 +118,31 @@ public async Task DeleteUser(int userId)
 
         public async Task<UserDTO> GetUserById(int userId)
         {
-            var user = await _context.Users.FirstOrDefaultAsync(p => p.Id == userId);
+            var user = await _context.Users
+                .Include(u => u.Teams)
+                    .ThenInclude(t => t.Users)
```

Comment:
Are you sure you need this second include?

Reply:
This is necessary to count the number of participants in the team